--- 1.8.2/Makefile.am.old	2005-08-04 18:25:24.000000000 +0900
+++ 1.8.2/Makefile.am	2005-08-04 18:29:10.000000000 +0900
@@ -16,7 +16,7 @@
 	cd $(top_srcdir) && $(MAKE) -f admin/Makefile.common subdirs
 
 $(top_srcdir)/acinclude.m4: $(top_srcdir)/admin/acinclude.m4.in $(top_srcdir)/admin/libtool.m4.in
-	@cd $(top_srcdir) && cat admin/acinclude.m4.in admin/libtool.m4.in > acinclude.m4
+	@cd $(top_srcdir) && cat admin/acinclude.m4.in admin/libtool.m4.in admin/ltsugar.m4.in admin/lt~obsolete.m4.in > acinclude.m4
 
 MAINTAINERCLEANFILES = subdirs configure.in acinclude.m4 configure.files 
 
